src/image.rs

```diff
@@ -495,33 +496,40 @@ pub trait GenericImage: Sized {
     }
 
     /// Returns a subimage that is a view into this image.
-    fn sub_image(&mut self, x: u32, y: u32, width: u32, height: u32) -> SubImage<Self>
-    where
-        Self: 'static,
-        <Self::Pixel as Pixel>::Subpixel: 'static,
-        Self::Pixel: 'static,
-    {
+    fn sub_image(&mut self, x: u32, y: u32, width: u32, height: u32)
+    -> SubImage<&mut Self>
+    where 
+		Self: 'static, 
+		<Self::Pixel as Pixel>::Subpixel: 'static,
+    	Self::Pixel: 'static 
+	{
```

Comment:
Can you move this `{` to the beginning of the line?

Reply:
I just copied that style from `copy_from` directly above. I also think it is slightly more readable due to the visible space between the type bounds and the function implementation.